src/ILLink.Tasks/build/Microsoft.NET.ILLink.targets

```diff
@@ -0,0 +1,365 @@
+<!--
```

Comment:
That would cause it to be imported automatically, which is not what I think we want (I think we need to control the targets import order relative to other targets in publish).

I'm not sure how best to communicate the naming decisions here.